app/editor/src/index.scss

```diff
@@ -94,26 +99,46 @@ h4 {
 // *************************************************
 .frm-in {
   padding-bottom: 0.5em;
+
   label {
     font-weight: bold;
   }
+
   &.chk {
     padding-bottom: 0em;
+
     label {
       cursor: pointer;
     }
   }
+
   &.chg {
     padding-bottom: 0;
+
     label:not(:first-child) {
       display: inline;
       font-weight: normal;
     }
   }
+
   &.rag {
     label:not(:first-child) {
       display: inline;
       font-weight: normal;
     }
   }
 }
+
+// *************************************************
+// Modal
+// *************************************************
+.modal {
```

Comment:
Needed this here because the modal is a root element.  It does not work within a component.